include/hc.hpp

```diff
@@ -23,6 +23,7 @@
 
 #include <hsa_atomic.h>
 #include <kalmar_cpu_launch.h>
+#include <hsa_ext_amd.h>
```

Comment:
This line marks a direct dependency from HC API to AMD HSA extension API, which is too strong a coupling, and would not be a good idea from API implementation point of view.